spec/datadog/core/configuration/components_spec.rb

```diff
@@ -1098,15 +1098,6 @@
         let(:info_response) { { endpoints: ['/v0.7/config'] }.to_json }
       end
 
-      context 'enabled' do
-        before { allow(settings.remote).to receive(:enabled).and_return(true) }
-
-        it 'starts the remote manager' do
-          expect(components.remote).to receive(:start)
-          startup!
-        end
-      end
```

Comment:
I wonder if we should have a negative test as well. I don't often advocate for those, but since this seems an important detail, it seems relevant to make sure a test fails if someone ever remembers to re-add something like the line that was removed?

Reply:
that would make sense

Reply:
Trying to reach green ASAP RN so I won't do that in this PR.

Reply:
Good call, not a blocker at all!